changes/af0603a5aef92f347e1d1c6a5292cddd.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: Deprecated network related API arguments for AHA APIs and related tools. Multi-network
```

Comment:
Maybe something more like `Deprecated support for multiple AHA network names being hosted by one AHA deployment. Synapse 3.0 AHA deployments will be simplified to hosting only one network.`